### DISCUSSION TITLE
[Help]: Errors while trying to install on ubuntu 23.04 arm64

### DISCUSSION THREAD

**Discussion by kjayGA** [mbentley/docker-omada-controller#382] [Help] [answered]
### Controller Version

v5.13.22

### Describe Your Issue or Question

When building the image receive errors noting 'Temporary failure resolving 'ports.ubuntu.com'
![image](https://github.com/mbentley/docker-omada-controller/assets/126358433/bf219709-f590-4922-8e26-35915b6d389f)

### Expected Behavior

Complete install

### Steps to Reproduce

sudo docker build --build-arg INSTALL_VER="5.13" -f Dockerfile.v5.x -t mbentley/omadacontroller:5.13 .

### How You're Launching the Container

```plain
I have not been able to build image to run
```

### Container Logs

```plain
there are no logs
```

### MongoDB Logs

```plain
there are no logs
```

**Comment by mbentley** [comment, marked as answer]
Looks like a DNS problem on your end based on the error.

**Reply by kjayGA** [reply]
Thanks
